Bvec bval

constellates opened the pull request:
Improvements for dwi, bval and bvec validation as per #55 . Definitely take a look at the additions to utils/issues.js because I made assumptions about the issues' severity and messages.

chrisfilo commented:
The error messages seem fine, but I found something else that was wrong.

As a test I mispelled the name of one of the files ( `mv dwi.bvec dwi.bvecs1`) in ds114 (it passes the validation without this change). I'm getting the following error:

	orange:bids-validator filo$ bin/bids-validator /Volumes/Samsung_T1/bids_examples/symlinked/ds114/
	/Users/filo/drive/workspace/bids-validator/validators/nii.js:29
	            bvec.split('\n')[0].replace(/^\s+|\s+$/g, '').split(' ').length, // bvec row 1 length
	                ^
	
	TypeError: Cannot read property 'split' of undefined
	    at NIFTI (/Users/filo/drive/workspace/bids-validator/validators/nii.js:29:17)
	    at /Users/filo/drive/workspace/bids-validator/validators/bids.js:171:25
	    at Gunzip.<anonymous> (/Users/filo/drive/workspace/bids-validator/utils/files.js:119:17)
	    at emitOne (events.js:78:13)
	    at Gunzip.emit (events.js:170:7)
	    at readableAddChunk (_stream_readable.js:147:16)
	    at Gunzip.Readable.push (_stream_readable.js:111:10)
	    at Gunzip.Transform.push (_stream_transform.js:129:32)
	    at Zlib.callback (zlib.js:573:14)

constellates commented:
I added two new errors (for missing .bvec and .bval files). I assumed that both .bvecs and .bvals are required for dwi scans. You should get a proper issue now instead of the unhandled error. Also added a test for this.
